photon-client/src/types/SettingTypes.ts

```diff
@@ -77,16 +78,60 @@ export interface VideoFormat {
   diagonalFOV?: number;
   horizontalFOV?: number;
   verticalFOV?: number;
-  standardDeviation?: number;
   mean?: number;
 }
 
+export enum CvType {
+  CV_8U = 0,
+  CV_8S = 1,
+  CV_16U = 2,
+  CV_16S = 3,
+  CV_32S = 4,
+  CV_32F = 5,
+  CV_64F = 6,
+  CV_16F = 7
+}
+
+export interface JsonMatOfDouble {
+  rows: number;
+  cols: number;
+  type: CvType;
+  data: number[];
+}
+
+export interface JsonImage {
+  rows: number;
+  cols: number;
+  type: CvType;
+  data: string; // base64 encoded
+}
+
+export interface CvPoint3 {
+  x: number;
+  y: number;
+  z: number;
+}
+export interface CvPoint {
+  x: number;
+  y: number;
+}
+
+export interface BoardObservation {
+  locationInObjectSpace: CvPoint3[];
+  locationInImageSpace: CvPoint[];
+  reprojectionErrors: CvPoint[];
+  optimisedCameraToObject: Pose3d;
+  includeObservationInCalibration: boolean;
+  snapshotName: string;
+  snapshotData: JsonImage;
+}
+
 export interface CameraCalibrationResult {
   resolution: Resolution;
-  distCoeffs: number[];
-  standardDeviation: number;
-  perViewErrors: number[];
-  intrinsics: number[];
+  cameraIntrinsics: JsonMatOfDouble;
+  // TODO rename to be Right
```

Comment:
wdym "Right"

Reply:
Should be renamed to distCoeffs. extrinsics are an unrelated thing

Reply:
Renamed